.eslintrc.js

```diff
@@ -2,16 +2,10 @@ module.exports = {
   extends: ['algolia', 'algolia/react'],
   rules: {
     'eslint-comments/disable-enable-pair': 0,
-    // Do not enforce a specific import order
-    'import/order': 0,
-    // Allow function components
-    'react/function-component-definition': 0,
```

Comment:
I think we should enforce function declarations everywhere.

Reply:
I agree, although the [default config](https://github.com/algolia/eslint-config-algolia/blob/master/packages/eslint-config-algolia/rules/react.js#L46-L51) sets a different preference. But this can be discussed another time. I updated the rule and the files in 32a7892.